### DISCUSSION TITLE
Problem with ThumpNails

### DISCUSSION THREAD

**Discussion by petzi69** [ImageMagick/ImageMagick#2463] [Help]
A few years ago I wrote a small program,  as far as ImageMagick is concerned, to crop a large number of screenshots, remove the background, make it transparent, reduce the image to thumpnail size, add a small shadow effect and save it as BMP with alpha channel. This worked fine back then, but unfortunately it doesn't work anymore, especially the target BMP doesn't have an alpha channel anymore, so there is no transparency. Instead, the backgrund ist black.

The ImageMagick command line is 

> convert input.bmp -crop 2560x1134+0+114 -fuzz 10% -trim -alpha set -alpha background -transparent #ec5b18 -thumbnail 89x17 -sharpen 0x2 ( -clone 0 -background black -shadow 80x1+1+1 ) +swap -background none -layers merge +repage -trim  -gravity east -extent 91x20  -define bmp3:alpha=true BMP3:output.bmp

When it worked, I used ImageMagick version 7.0.3-5 on Windows 10 with an Intel CPU and NVidia graphics card, now version 7.0.10-27 on Windows with an AMD CPU and AMD graphics card, if that matters. I already tried with the old ImageMagick version from back then, but the result doesn't change.

What am I doing wrong?

**Comment by snibgo** [comment]
If you use Windows BAT, the % should be doubled, to "10%%".

I suggest you insert "+repage" after the "-crop", and after the "-trim".

As you use v7, I suggest you use "magick", not "convert", and not "magick convert".

But your command seems to work for me with IM v7.0.10-27, and the output has transparency. Please post a sample input.bmp, and the resulting output.bmp.

**Comment by petzi69** [comment]
I tried it again, changing "convert" to "Magick" probably did the trick! Thanks for the advice!
